tools/junitreport/README.md

```diff
@@ -0,0 +1,47 @@
+# junitreport
```

Comment:
You are no longer supposed to vendor this

Reply:
Junk some SSCS.